### PR TITLE
blacken docs

### PR DESCRIPTION
**Pull request by asottile**
Hi friends, I made a terrific (terrible? terrifying?) thing.

First commit is manual changes:
- tabs to spaces
- needed to add some `...`s in a few places to make valid syntax

Second commit is automated!

Using the new thing I made today: https://github.com/asottile/blacken-docs

Heavily inspired by #3525.  See also ambv/black#292, ambv/black#294.  CC @blueyed

### CODE REVIEW

**Comment by RonnyPfannschmidt** [comment]
@asottile it seems its broken on windows ^^

**Comment by asottile** [comment]
/me shakes fists at charmap

easy fix

File: doc/en/bash-completion.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ Install argcomplete using::
 
 For global activation of all argcomplete enabled python applications run::
 
-	sudo activate-global-python-argcomplete
+    sudo activate-global-python-argcomplete
```
Comment (ambv) [inline doc/en/bash-completion.rst]: That looks like a false positive, no?

**Comment by asottile** [comment]
@ambv if you read the description, I split it up into two commits (one manual, one automatic) -- among the first commit:

> tabs to spaces

**Comment by ambv** [comment]
Sorry, got confused by this.

ok-then

**Pull request merged**